### PR TITLE
Add walk-cross-device to walk policy

### PR DESCRIPTION
**Pull request by redgoat650**
Should fix issue seen in shippable testing, where files were skipped in the walk for being cross-device.

### CODE REVIEW

File: tests/tools/fswalker/walker/walker.go [hunk 1]
```diff
@@ -65,5 +65,6 @@ func WalkPathHash(ctx context.Context, path string) (*fspb.Walk, error) {
 		Include:         []string{path},
 		HashPfx:         []string{""}, // Hash everything
 		MaxHashFileSize: MaxFileSizeToHash,
+		WalkCrossDevice: true,
```
Comment (julio-lopez) [inline tests/tools/fswalker/walker/walker.go]: This permits the walk to cross device (or probably mount-point) boundaries.
Was the error during the walk or when comparing the reports?

Comment (julio-lopez) [reply tests/tools/fswalker/walker/walker.go]: From the error message, it appears to be during the walk

Comment (redgoat650) [reply tests/tools/fswalker/walker/walker.go]: Was during the walk, here:
https://github.com/google/fswalker/blob/7aa03fed36caf47cb8b384ef400778ef73f09db6/walker.go#L289

**Comment by julio-lopez** [comment]
This may be due to running in a docker container where a FS tree is made of many overlays.

May want to include that ^^^ when submitting upstream

LGTM  ✅ 
🚢it :shipit:

**Pull request merged**